### DISCUSSION TITLE
GPU Utilisation only 7-10%

### DISCUSSION THREAD

**Discussion by Toolfolks** [SWivid/F5-TTS#163] [Ideas]
I am running on NVIDIA 3060. When producing the audio file it uses between 7-10% of the GPU. 

Any 'magic' switches or code changes to speed the process. I changed nfe_step = 16 and this was faster but takes about 8 secs to produce an 8 sec audio.
Using threads ???
Does it re-use the sample audio on each subsequent synthesis ( could it keep this in memory )
 .
 Any help appreciated.

**Comment by Toolfolks** [comment]
Also I see Generating audio using F5-TTS in 2 batches. Can we use more batches at once.?

**Reply by Chipstix24** [reply]
I'm having a similar issue, where the first run will be about 30mins (M1 Macbook with built in GPU). Then I re-run with same reference audio and new text and it has taken 8 hours to get to 44%?! Not sure if should be clearing cache, or restarting my machine between each run for example?
